gamedata/scripts/magazines.script

```diff
@@ -534,6 +534,7 @@ function load_magazine(magazine, capacity, delay)
 
 	local tg = time_global()
 	if tg < last_time + delay then return false end
+	if IsMoveState("mcAnyMove") and tg < last_time + (3 * delay) then return false end
```

Comment:
line above will always return false and break the loop before this line gets to.

Reply:
If tg is between (last_time + delay) and (last_time + delay * 3) it will pass the first condition due to being greater, then fail the second condition.